x/profile/types/message_update_validator_description.go

```diff
@@ -0,0 +1,57 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+var _ sdk.Msg = &MsgUpdateValidatorDescription{}
+
+func NewMsgUpdateValidatorDescription(
+	address string,
+	identity string,
+	moniker string,
+	website string,
+	securityContact string,
+	details string,
+) *MsgUpdateValidatorDescription {
+	return &MsgUpdateValidatorDescription{
+		Address: address,
+		Description: &ValidatorDescription{
+			Identity:        identity,
+			Moniker:         moniker,
+			Website:         website,
+			SecurityContact: securityContact,
+			Details:         details,
+		},
+	}
+}
+
+func (msg *MsgUpdateValidatorDescription) Route() string {
+	return RouterKey
+}
+
+func (msg *MsgUpdateValidatorDescription) Type() string {
+	return "UpdateValidatorDescription"
+}
+
+func (msg *MsgUpdateValidatorDescription) GetSigners() []sdk.AccAddress {
+	address, err := sdk.AccAddressFromBech32(msg.Address)
+	if err != nil {
+		panic(err)
+	}
+	return []sdk.AccAddress{address}
+}
+
+func (msg *MsgUpdateValidatorDescription) GetSignBytes() []byte {
+	bz := ModuleCdc.MustMarshalJSON(msg)
+	return sdk.MustSortJSON(bz)
+}
+
+func (msg *MsgUpdateValidatorDescription) ValidateBasic() error {
```

Comment:
Another check we can have is to check if at least one field of the description is defined.

The benefits of `ValidateBasic()` is that the message is never spread over the network if it fails and so we can avoid to spread a message that has no side effect